resources/perform-backup.sh

```
@@ -84,6 +84,12 @@ if [ "$has_failed" = false ]; then
             fi
             rm /tmp/"$DUMP"
 
+            # Check if the uploaded file is empty and set has_failed to true if it is
+            if [ $(stat -c%s "/tmp/$DUMP") -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)